draft-irtf-cfrg-aegis-aead.md

```diff
@@ -1616,17 +1616,17 @@ stream, tag = Encrypt(ZeroPad({}, len), {}, key, nonce)
 return stream
 ~~~
 
-This is equivalent to encrypting a `len` all-zero bits message without associated data, and discarding the authentication tag.
+This is equivalent to encrypting a `len` all-zero bits message without associated data and discarding the authentication tag.
 
-Instead of relying on the generic `Encrypt` function, implementations can skip the finalization step.
+Instead of relying on the generic `Encrypt` function, implementations can omit the `Finalize` function.
 
 After initialization, the `Update` function is called with constant parameters, allowing further optimizations.
```

Comment:
What about `allowing values to be cached` or similar?

Reply:
See my answer in the issues. Many different types of optimizations can happen. Optimizing compilers are likely to completely remove the XORs rather than caching zeros. Users can also manually write a specialized `Enc` function if that can help (or in assembly). I think it's better to just hint at the fact that it enables optimizations.